terminal/command.go

```diff
@@ -724,7 +726,11 @@ func printcontext(state *api.DebuggerState) error {
 	}
 	if len(state.CurrentThread.File) == 0 {
 		fmt.Printf("Stopped at: 0x%x\n", state.CurrentThread.PC)
-		fmt.Printf("\033[34m=>\033[0m    no source available\n")
+		if dumbTerminal {
```

Comment:
It makes sense, as it will allow for further improvements / usages as well. I'll try to do the refactoring tomorrow. Thanks for feedback.